go-sdk/Makefile

```diff
@@ -11,7 +11,7 @@ GOGC=30
 
 all: test go-mod-tidy
 test: deps lint
-		$(VGO) test ./internal/... ./pkg/... -cover -coverprofile=coverage.txt -covermode=atomic -timeout=30s ${TEST_ARGS}
```

Comment:
As the pkg folder now only contains interfaces and simple wrapper code, it does seem if we have tests in this folder, they'll mostly be boiler plate code.

As long as we can ensure other contributors know this convention, we should be fine.